src/ethereum_spec_tools/evm_tools/t8n/t8n_types.py

```diff
@@ -178,7 +178,7 @@ def parse_json_tx(self, raw_tx: Any) -> Any:
 
         # tf tool might provide None instead of 0
         # for v, r, s
-        raw_tx["v"] = raw_tx.get("v") or "0x00"
+        raw_tx["v"] = raw_tx.get("v") or "0x00" or raw_tx.get("y_parity")
```

Comment:
`"0x00"` is always truthy, so you probably want:

```suggestion
        raw_tx["v"] = raw_tx.get("v") or raw_tx.get("y_parity") or "0x00"
```